crates/uv/src/commands/project/run.rs

```diff
@@ -251,7 +251,7 @@ pub(crate) async fn run(
                     lock: &lock,
                 },
                 &ExtrasSpecification::default(),
-                &DevGroupsManifest::default(),
+                &DevGroupsSpecification::default().with_defaults(Vec::new()),
```

Comment:
This looks a little weird?

Reply:
I agree it's a bit weird, but on the other hand I do kind of like that it's really hammering in "there were two different things I was theoretically supposed to be inputting and I'm doing Neither of them".